src/main/java/anab/Person/ManagePerson.java

```diff
@@ -1,34 +1,122 @@
+        trainer.addFeedbackMark("Ionica", 8);
+        trainer.addFeedbackMark(2);
+        trainer.addFeedbackMark(3);
+        trainer.addFeedbackMark(4);
+        trainer.addFeedbackMark(5);
+        trainer.addFeedbackMark(6);
+        trainer.addFeedbackMark(7);
+        trainer.addFeedbackMark("Fanel", 10);
+        trainer.addFeedbackMark("Vasile", 7);
+      /*  feedbackMarks.put("Ionica", 8);
+        feedbackMarks.put("None", 2);
+        feedbackMarks.put("Fanel", 10);
+        feedbackMarks.put("Vasile", 7);
+       */
+        trainer.getFeedbackMarks();
+        trainer.printFeedbackMarks();
+        trainer.printPersonDetails();
+        System.out.println("Average feedback for " + trainer.getLastName() + "= " + trainer.averageFeedback());
+        System.out.println(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>....1......>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");
+        Map<String, Integer> feedbackMarks1 = new HashMap<String, Integer>();
+        Trainer trainer1 = new Trainer("Eugenius ", "Smartus ", "Male", 30, "eug.Sm@domain.com", false, "Istorie", 5, feedbackMarks1);
+        System.out.println(trainer1.getFirstName() + trainer1.getLastName() + "trainer " + trainer1.getSpecialization() + " - actual experience: " + trainer1.getExperienceYears() + " ani");
+        System.out.println("New experience added for trainer " + trainer1.getLastName() + " = " + trainer1.getExperienceYears() + " years");
+        System.out.println("Total experienced years= " + trainer1.getExperienceYears());
+        trainer1.setSpecialization("Istorie");
+        trainer1.setExperienceYears(5);
+        trainer1.addFeedbackMark("Ionica", 3);
+        trainer1.addFeedbackMark(3);
+        trainer1.addFeedbackMark("Fanel", 3);
+        trainer1.addFeedbackMark("Vasile", 3);
+        /*feedbackMarks1.put("Ionica", 3);
+        feedbackMarks1.put("Test",3);
+        feedbackMarks1.put("Fanel", 3);
+        feedbackMarks1.put("Vasile", 3);
+        */
+        trainer1.getFeedbackMarks();
+        trainer1.printFeedbackMarks();
+        trainer1.printPersonDetails();
+        System.out.println("Average feedback for " + trainer1.getLastName() + "= " + trainer1.averageFeedback());
+        System.out.println(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>....2......>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");
+        Map<String, Integer> feedbackMarks2 = new HashMap<String, Integer>();
+        Trainer trainer2 = new Trainer("George ", "Thomas ", "GenderFluid", 50, "smileyDude@fun.com", false, "Engleza", 12, feedbackMarks2);
+        System.out.println(trainer2.getFirstName() + trainer2.getLastName() + "trainer " + trainer2.getSpecialization() + " - actual experience: " + trainer2.getExperienceYears() + " ani");
+        System.out.println("New experience added for trainer " + trainer2.getLastName() + " = " + trainer2.getExperienceYears() + " years");
+        System.out.println("Total experienced years= " + trainer2.getExperienceYears());
```

Comment:
you can use `printFeedbackDetails()` to print the trainer's details

Reply:
Replaced `System.out.println()` with the function `printPersonDetails`